packages/x-components/tests/e2e/preselected-filters.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Preselected filters
+
+  Background:
+    Given a next queries API
+    And   a suggestions API
+    And   a related tags API
+    And   a tracking API
+    And   a recommendations API with a known response
+    And   a results API with a known response
+
+  Scenario Outline: 1. Preselected filter are selected is there is no more filters in the url
+    Given   an application the "<filter>" filter preselected
+    When  start button is clicked
+    And  "<query>" is searched
+    Then  related results are displayed
+    And   url contains parameter "filter" with value "<filter>"
+    And filters "<filter>" are shown in the selected filters
```

Comment:
I hate this code style, but format the steps in both scenarios so they are aligned.